envoy/tests/legacy/test_utils.py

```diff
@@ -74,6 +74,20 @@ def test_wildcard_removal_tree():
                 ()
             ),
             "method": "monotonic_count",
+        },
+        "connection_limit.*.active_connections": {
+            "tags": (
+                ("stat_prefix",),
+                ()
+            ),
+            "method": "monotonic_count",
+        },
+        "connection_limit.*.limited_connections": {
+            "tags": (
+                ("stat_prefix",),
+                ()
+            ),
+            "method": "monotonic_count",
         }
```

Comment:
I think this can be removed since the metric matcher doesn't include the wildcard anymore.